infra/main.tf

```diff
@@ -449,7 +462,29 @@ module "ci_gsa_secret" {
   name = "ci"
 }
 
-resource "google_artifact_registry_repository_iam_member" "artifact_registry_viewer" {
+resource "google_storage_bucket_iam_member" "ci_bucket_viewer" {
+  bucket = module.hail_ci_bucket.name
+  role = "roles/storage.objectAdmin"
+  member = "serviceAccount:${module.ci_gsa_secret.email}"
+}
+
+resource "google_storage_bucket_iam_member" "gcr_ci_viewer" {
+  bucket = google_container_registry.registry.id
+  role = "roles/storage.objectViewer"
+  member = "serviceAccount:${module.ci_gsa_secret.email}"
+}
+
+resource "kubernetes_secret" "hail_ci_0_1_service_account_key" {
+  metadata {
```

Comment:
I've always found the name of this confusing.

Reply:
Me too